hathor/indexes/manager.py

```diff
@@ -46,6 +47,7 @@ class IndexesManager(ABC):
     sorted_blocks: TimestampIndex
     sorted_txs: TimestampIndex
 
+    mempool_tips: MempoolTipsIndex
```

Comment:
Maybe `MempoolIndex` is better, but it actually only stores the tips. It can iterate more than just the tips if we pass the storage to it so it can make a walk (on iter_all).

Reply:
Ah, makes sense. We can keep `MempoolTipsIndex` given that iter_all is only with storage